docs/content/en/docs/compatibility/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Compatibility"
+weight: 1
+description: >
+  Fission Compatibility with Keda and Keda Connectors
+---
+
+| Fission version | Keda version | Keda Connectors image tag |
+| --------------- | ------------ | ------------------------- |
+| 1.11.2          | 1.5          | v0.1                      |
+| 1.12.0          | 2.0          | v0.5                      |
+
+All Keda Connector images can be used with both fission version except the Kafka connector, which has some breaking changes.
```

Comment:
Should we also add a few lines about the fact that Environment images will follow a different versioning scheme going forward?

Reply:
Yes, will add it